.vscode/settings.json

```diff
@@ -2,17 +2,9 @@
   "editor.formatOnSave": true,
   "editor.tabSize": 2,
   "editor.insertSpaces": true,
-  "prettier.disableLanguages": ["vue", "markdown"],
   "files.insertFinalNewline": true,
-  "files.exclude": {
-    "**/src/**/*.css": true
-  },
-  "search.exclude": {
-    "**/src/**/*.css": true
-  },
-  "files.watcherExclude": {
-    "**/lib/**": true,
-    "**/src/**/*.css": true
+  "[markdown]": {
+    "editor.defaultFormatter": "vscode.markdown-language-features"
```

Comment:
sure about the removed props? None of them are needed anymore?

Reply:
i didnt see any difference

Reply:
I've made prettier as default formatter for all extensions except markdown. Seems to work fine